docs/src/operations/metrics.md

```diff
@@ -29,44 +29,23 @@ Metrics can be configured in the [configuration file](operations/the-zeebecfgtom
 
 ## Connecting Prometheus
 
-As explained, Zeebe writes metrics to a file. The default location of the file is `$ZB_HOME/metrics/zeebe.prom`. There are two ways to connect Zeebe to Prometheus:
+As explained, Zeebe exposes the metrics over a HTTP server. The default port is `26000`.
```

Comment:
Is this port different from MetricsCfg.DEFAULT_PORT (=9600)?

Reply:
Sorry my fault. I mixed it up with our other ports :D I will fix it